### ISSUE TITLE
This topic is closed.  Do not open further issues on this topic.

### ISSUE THREAD

**Issue by amidevous** [rocky-linux/rocky-tools#211]
This topic is closed.  Do not open further issues on this topic.

_Originally posted by @pajamian in https://github.com/rocky-linux/rocky-tools/issues/210#issuecomment-1936977544_
            

are you stupid or what I'm coming to tell you if you don't have the answer and you leave it open and you just don't answer

it's a lack of respect to close tickets instead of those who open them, you don't know, you don't answer and you let who else who has the answer answer, period

but in no case do you have the right to close the exit for me, it's a total lack of respect

all I'm asking is how to migrate from Ferora point
it's not because your tool doesn't allow you to do it that it gives you the right to close the issue
there must be a method to do it manually
what I'm asking is like everyone else
if you don't have the answer, you leave no point
just because you have access to close the exits does not give you the right to do so
the only one who must decide if an exit must be closed is the one who opened it
You have the right to close only if the subject is resolved and the one who opened the issue forgot to do so.
if you do it for another reason it's disrespectful to the person making the request
There's no point in closing every time you close
I will open a new one and if I have to spend my life there to get the answer I will do so I can guarantee that if you continue to close without an answer that I consider acceptable being given
I will find you anywhere in the world and you will no longer recognize your face
Basically I'm not disrespecting you
it was you who came to disrespect me by closing my request twice without it being
freedom of expression applies to everyone and if you don't know and change country

if you don't know you leave the outcome you leave it open point
there is bound to be someone else who will have the answer

**Comment by nazunalika** [comment]
The scripts are designed to migrate from one Enterprise Linux to Rocky Linux, that is all. The reality is that Fedora is far too ahead of Rocky Linux and other Enterprise Linux distributions. If you compare the package versions, you'll end up seeing that it does not make sense to ever attempt such a migration. This is especially the case with packages like glibc, rpm, systemd, kernel, grub2 where even if the versions look so minor between the two, so much has changed that swapping the packages can lead to a system being unbootable and unusable.

```
[root@botsystem ~]# rpm -q glibc rpm
glibc-2.38-16.fc39.x86_64
rpm-4.19.1-2.fc39.x86_64

[root@xmpp01 host]# rpm -q rpm glibc
glibc-2.34-83.el9.7.x86_64
rpm-4.16.1.3-27.el9_3.x86_64
```

You have been asked to stop opening posts on migrating from Fedora to Rocky Linux. This issue is not closed.
